src/js/player.js

```diff
@@ -502,7 +502,8 @@ class Player extends Component {
       'autoplay': this.options_.autoplay,
       'preload': this.options_.preload,
       'loop': this.options_.loop,
-      'muted': this.options_.muted
+      'muted': this.options_.muted,
+      'language': this.options_.language
```

Comment:
You might want to use the player's language() getter here. It would probably provide some more consistency.